app/models/exercise.rb

```diff
@@ -2,6 +2,7 @@ class Exercise < ApplicationRecord
   validates :name, presence: true
   validates :content, presence: true
   validates :description, presence: true
+  validates :tags, presence: true
```

Comment:
I dig it for now, but for the future do we want to allow posts to not have tags? (Is that already a thing? If so: can't brain today. 😬 )